linux_os/guide/system/network/network-nftables/nftables_ensure_default_deny_policy/rule.yml

```diff
@@ -15,7 +15,7 @@ rationale: |-
 
 severity: medium
 
-platform: package[nftables]
+platform: package[nftables] and not package[firewalld]
```

Comment:
In most cases, both packages are installed but only one is enabled/started. I don't think it is a good idea to make them mutually exclusive. But including `package[nftables]` sounds reasonable to me.

Reply:
The idea here is that the check from my perspective should be performed when we have only nftables and firewall is managed through the nftables service itself, because if we have firewalld the configuration for default policy will be done through firewalld configuration

Reply:
This is exactly what I was trying to avoid - to have different profiles depending on what package user has installed. The situation is that currently we might end up with most of the distros have any combination of firewalld, nftables, iptables, and using the package platform seemed like the only solution to go with so the user has the freedom to install any package and still be compliant with the profile that covers all.

Reply:
The applicability would work better on the service level instead of package level, like in https://github.com/ComplianceAsCode/content/pull/10812.